src/__test-utils__/createTestApp.ts

```
@@ -0,0 +1,70 @@
+import type { AddressInfo } from 'net';
+import cookieParser from 'cookie-parser';
+import httpContext from 'express-http-context';
+import identityMiddleware from '../middleware/identity-middleware';
+
+export interface TestResponse {
+  status: number;
+  body: string;
+  headers: http.IncomingHttpHeaders;
+}
+
+export function createTestApp(...additionalMiddleware: RequestHandler[]): Express {
```

ESLint

[error] Replace `...additionalMiddleware:·RequestHandler[]` with `⏎··...additionalMiddleware:·RequestHandler[]⏎`

(prettier/prettier)